src/main/kotlin/com/openlattice/mechanic/upgrades/MediaServerUpgrade.kt

```diff
@@ -0,0 +1,118 @@
+package com.openlattice.mechanic.upgrades
+
+import com.openlattice.ResourceConfigurationLoader
+import com.openlattice.data.storage.ByteBlobDataManager
+import com.openlattice.data.storage.LocalAwsBlobDataService
+import com.openlattice.datastore.configuration.DatastoreConfiguration
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.postgres.DataTables.quote
+
+//Migration for media server
+
+private val BINARY_PROPERTY_ID_TO_FQN = mapOf("90abc3fb-f0c4-45e5-b385-8cf70c06ef81" to "ol.digitalsignature",
+        "881c2601-f3b9-40cd-bb9b-86e811182156" to "ol.licenseplateimage",
+        "faf3c011-b386-4c51-a1bb-4bad27b4083f" to "ol.pictureback",
+        "e8174f66-4b80-4f8c-bcd3-2924d5ccd6d7" to "ol.picturefront",
+        "0f95049c-b668-49b1-bf32-f26835471b0a" to "ol.vehicleimage",
+        "45aa6695-a7e7-46b6-96bd-782e6aa9ac13" to "publicsafety.mugshot"
+        )
+
+private val ENTITY_SET_ID = quote("entity_set_id")
+private val ID = quote("id")
+
+class MediaServerUpgrade(private val toolbox: Toolbox) : Upgrade {
+    private lateinit var byteBlobDataManager: ByteBlobDataManager
+
+    override fun upgrade(): Boolean {
+        setUp()
+        addMockS3Table()
+        migrateBinaryProperties()
+        return true
+    }
+
+    override fun getSupportedVersion(): Long {
+        return Version.V2018_09_14.value
+    }
+
+    fun setUp() {
+        val config = ResourceConfigurationLoader.loadConfiguration(DatastoreConfiguration::class.java)
+        val byteBlobDataManager = LocalAwsBlobDataService(config)
+        this.byteBlobDataManager = byteBlobDataManager
+    }
+
+    fun addMockS3Table() {
```

Comment:
What is this for?

Reply:
For the medialocal profile. For testing an integration without using the s3 bucket we talked about having a table in postgres to write to before testing the integration using the s3 bucket